.danger/index.ts

```diff
@@ -35,16 +35,20 @@ const isCurrentDevelopOrMain = ['main', 'develop'].includes(currentBranch);
 const isTargetBranchDevelopOrMain = ['main', 'develop'].includes(targetBranch);
 
 // validate if the current and target branch are specifically main or develop
-const isTargetBranchMain = targetBranch === 'main';
 const isCurrentDevelop = currentBranch === 'develop';
+const isTargetBranchMain = targetBranch === 'main';
+
+// [FDS-444]: identify release branches and PRs
+const isCurrentAReleaseBranch = currentBranch.search(/^release/) !== -1;
+const isReleasePR = isCurrentAReleaseBranch && isTargetBranchMain;
 
 const shouldDangerCheckPR =
-  isCurrentDevelopOrMain !== isTargetBranchDevelopOrMain;
+  isCurrentDevelopOrMain && isTargetBranchDevelopOrMain;
```

Comment:
@rociobaena I think we are almost there, the AND operator is ok but we still need to negate the values of `isCurrentDevelopOrMain` and `isTargetBranchDevelopOrMain `.

Reply:
```suggestion
  !isCurrentDevelopOrMain && !isTargetBranchDevelopOrMain;
```

Our goal is to make Danger.js skip its checks if origin branch is `Develop` and target branch is `Main`.

Reply:
Manu, thank you for reviewing this. But please wait for that, right now I'm checking the fix works for all scenarios. I would like not to have you as a review for now, but you're for default, so.. just wait for me :D

Reply:
@lordpixel , seems the code is working now. Please take a look again.